LICENSE

```diff
@@ -1,7 +1,7 @@
 BSD 3-Clause License
 
-British Crown copyright. The Met Office.
-All rights reserved.
+Copyright (c) 2022, The Met Office.
+git
```

Comment:
```suggestion
```
Does the standard github BSD 3-Clause License add "git" like this here?

Reply:
Sorry!🙆 My mistake
Let me resolve it